packages/fxa-settings/src/pages/InlineRecoveryKeySetup/index.tsx

```diff
@@ -79,8 +99,8 @@ export const InlineRecoveryKeySetup = ({
         return (
           <InlineRecoveryKeySetupCreate
             {...{ createRecoveryKeyHandler }}
```

Comment:
Could use syntactic sugar here
```suggestion
            {...{ createRecoveryKeyHandler, doLaterHandler }}
```